admin_guide/managing_networking.adoc

```diff
@@ -1147,3 +1147,50 @@ objects:
       - podSelector: {}
 ...
 ----
+
+[[admin-guide-enabling-hsts]]
+== Enabling HTTP Strict Transport Security
+
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
+ensures traffic to a host will always be HTTPS. This is useful for ensuring
```

Comment:
s/ensures traffic will always be HTTPS/requests traffic to the host to be HTTPS/

https tells the client, in a HTTPS response, that future requests should be HTTPS. The client decides whether or not to honor the request. So "ensures" is a little strong.